plugins/modules/zos_copy.py

```diff
@@ -563,6 +563,61 @@
     returned: success
     type: str
     sample: SAMPLE.SEQ.DATA.SET
+dest_created:
+    description: Boolean value that indicate the creation of dest.
+    returned: success and if dest is create by the module.
+    type: bool
+    sample: true
+destination_attributes:
+    description: Values of a dest create by the module.
+    returned: success and values of the dest created by module.
+    type: dict
+    contains:
+      block_size:
+        description:
+          Block size of the dataset.
+        type: int
+        sample: 32760
+      record_format:
+        description:
+          Type of record neither binary or not.
+        type: str
+        sample: FB
+      record_length:
+        description:
+          Length of the dataset.
+        type: int
+        sample: 45
+      space_primary:
```

Comment:
For `space_primary` and `space_secondary`, I think it would be better to say "Allocated space for the dataset" instead of "Needed size..."

I don't understand well what you mean in `space_type`, but I think you meant to say something like "Unit of measurement for space".